assets/src/apps/authoring/components/AdaptivityEditor/ConditionItemEditor.tsx

```diff
@@ -171,7 +177,11 @@ const ConditionItemEditor: React.FC<ConditionItemEditorProps> = (props) => {
         key={`value-${parentIndex}`}
         id={`value-${parentIndex}`}
         defaultValue={value}
-        onBlur={(e) => handleValueChange(e)}
+        onBlur={(e) => {
```

Comment:
If user is trying to modify / delete a condition value, we should not trigger the part component delete prompt.